### ISSUE TITLE
show unweighted data is biased

### ISSUE THREAD

**Issue by timbender-ncceh** [timbender-ncceh/ce_assessment#5]
send to Andrea charts or visuals how data 
* [ ] unweighted data is biased for total survey
* [ ] unweighted data is biased for top 20% of scores 

Due 8/10/23

**Comment by timbender-ncceh** [comment]
![image](https://github.com/timbender-ncceh/ce_assessment/assets/105810134/f57fd1aa-a115-4443-8bbf-79ca3cd19d16)

**Comment by timbender-ncceh** [comment]
![image](https://github.com/timbender-ncceh/ce_assessment/assets/105810134/82c92e45-b44a-408f-b478-b245915c5719)

**Comment by andreaNCCEH** [comment]
Sorry, to clarify: I was really hoping for a just a summary of the unweighted (i.e. not pushed to counter-act any bias) of the untouched scores and distribution. 

I think that what these charts show that you made during the meeting? 

![image (48)](https://github.com/timbender-ncceh/ce_assessment/assets/67340114/6ebaf59d-0b0d-4e08-8b5e-9cdf58324521)
![image (49)](https://github.com/timbender-ncceh/ce_assessment/assets/67340114/d4f46de1-b34b-4f1a-82e0-3e86af0c0704)

**Comment by timbender-ncceh** [comment]
Tangentially related to bias, I ran the model with some weights that came from 1 simulation that seemed like a good fit and then came across this plot: 
![image](https://github.com/timbender-ncceh/ce_assessment/assets/105810134/c23ebf26-2e81-4af4-9c28-0bdb7f3486cd)
It shows that the makeup of Black hit our goal, but White also increased (and pretty much ate up all the other races).  

Time to put on thinking caps @andreaNCCEH

**Comment by timbender-ncceh** [comment]
> send to Andrea charts or visuals how data
> 
> * [ ]  unweighted data is biased for total survey
> * [ ]  unweighted data is biased for top 20% of scores
> 
> Due 8/10/23

@andreaNCCEH I was looking through open issues this morning and saw this and have a couple thoughts I'd like to send your way for consideration:  

1. You pointed out that the bell curve charts I posted [here](https://github.com/timbender-ncceh/ce_assessment/issues/5#issuecomment-1672110134), and [here](https://github.com/timbender-ncceh/ce_assessment/issues/5#issuecomment-1672110943) were confusing and I agree.  Looking now I can't quickly grasp the message I was trying to communicate.  
2. There are a lot of tasks in this issue that are seemingly unrelated, and i think it makes sense to split the ones that remain into new issues and close this one out.  
  - See [this](https://github.com/timbender-ncceh/ce_assessment/issues/5#issuecomment-1673596816) comment about boxplots
3. The rest of the comments made here have been addressed by #6 (now closed) and #2 (now closed). 

So here's my recommendation for this issue
- [ ] Create a new issue for the boxplot comment above and make it a part of the summary output for the README, 
- [ ] Close out this issue.
